src/uwtools/apps/srw.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+#pylint: disable=unused-import, unused-variable, unused-argument, useless-parent-delegation
+# remove these disables once implemented
+'''
+This file contains the specific drivers for a particular app, using the facade pattern base class.
+'''
+
+import logging
+import os
+import sys
+import shutil
+import subprocess
+
+from uwtools.utils import file_helpers
+from uwtools.utils import cli_helpers
+from ..drivers.facade import Facade
+
+logging.getLogger(__name__)
+
+
+class SRW210(Facade): # pragma: no cover
+    #remove pragma when completed
+
+    '''
+        Concrete class to handle UFS Short Range Weather app forecasts.
+    '''
+    def __init__(self, argv):
+
+        '''
+        Initialize the facade driver.
+
+        '''
+        super().__init__(argv)
+
+    def load_config(self, config_file):
+        '''
+        Load the configuration file.
+
+        '''
+        file_type = cli_helpers.get_file_type(config_file)
+        if file_type == 'INI':
+            with open('config.yaml', 'w', encoding="utf-8") as file_name:
+                ## Note: this is a temporary path until parsing the SRW directory is implemented
+                subprocess.call(["python", "config_utils.py", "-c", config_file, "-t",
+                             "$PWD/config_defaults.yaml", "-o", "yaml"], stdout=file_name)
```

Comment:
1) Yes, this is the current SRW config method. Strictly including it explicitly here is out of scope, but I found it helpful for reference.
2) True! It may just still be an old habit to overly describe placeholders
3) Actually, both are new to me, so any more clear techniques are appreciated!

Reply:
I mostly like `subprocess.run()` and `.check_output()` because you can just write out the command you're executing instead of breaking it down into an array, which is, for me, harder to read/understand.

Reply:
"invalid name 'f'" c'mon pylint, sheesh